source/documentation/deploying-an-app/use-circleci-to-upgrade-app.html.md.erb

```diff
@@ -71,62 +161,98 @@ To authenticate with ECR, you will need to set:
-                          --set image.tag="${CIRCLE_SHA1}" \
-                          --set deploy.host="${APPLICATION_HOST_URL}" \
-                          --set replicaCount="4" \
-                          --debug
-          fi
+          if [ "${CIRCLE_BRANCH}" == "master" ]; th en
+            helm upgrade demo helm_deploy --install --namespace=${K8S_NAMESPACE}
+          fi 
+          kubectl --namespace=${K8S_NAMESPACE} get pods
 ```
 
-[deploy-helm]: /documentation/deploying-an-app/app-deploy-helm.html#deploying-an-application-to-the-cloud-platform-with-helm
+In the above config.yml replace CONTEXT with the name of the context you created. 
+Also replace REPO-NAME with the name of your ECR repo proceeding the tag name. For example if my repo name is my-repo/test and my image tag is rails then I would replace
+REPO-NAME with my-repo/test:rails
+
+The above config.yml needs to be saved in a folder called .circleci in the root of your application repository. 
+For demo purposes you may use the [multi-demo container repository] (https://user-guide.cloud-platform.service.justice.gov.uk/documentation/deploying-an-app/multicontainer-app-deploy.html)
+
+In the 'jobs' section we have a 'docker' executor where we will execute the steps inside the specified docker image. In this case we will run our pipeline steps inside a circle ci docker image.
+
+We then have 3 key steps as follows:
+
+1) Build / Push our application docker image. 
+
+   The multi-container demo repo has a folder called 'rails-app'. Inside this folder is a Dockerfile that defines the rail app. We run 'docker build..'
+   inside this directory. The next command is to push the image into your ECR 
+
+2) Authenticate with cluster
+
+In this step we issue a series of 'kubectl' commands to authenticate against the cluster, scoped to the namespace to which the service account belongs. 
+Earlier above we added the ${K8S_CLUSTER_CERT} and ${K8S_TOKEN} variables into our context in the circle ci console. These variables are injected into the circle CI image.
+The kubectl commands can then use these env variables to authenticate against the cluster. 
+
+
+3) Run Helm Chart 
+
+In the multi-container repo there is folder that contains the helm chart for the rails demo app. This step will run the helm charts and deploy the frontend.
+
+### Triggering the Pipeline
+
+The pipeline will automatically be triggered when a change to the 'master' is commmited. Circle CI will execute the latest config.yml and execute each step in the pipeline.
+
+Once all the above is complete make a change to the rails app and commit to master. You should now see a new job creaed in Circle CI 
+
+In Circle CI click on 'Jobs' > Click on your repo > Click on the new job > You shoud now see all your steps in the dashboard and their status
+
+After the job has finished your rails app should now be up and running. You can access the app by hitting the DNS you specified in your helm chart values for the rails app.
```

Comment:
I'd suggest "visiting the URL" instead of "hitting the DNS"

Reply:
done

Reply:
done.